src/com/powem/inv/algos/NetworkResilienceChecker.java

```diff
@@ -0,0 +1,130 @@
+package com.powem.inv.algos;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+
+//        Problem: Network Connectivity Check
```

Comment:
These problems are starting to look all too similar. Need to branch out to non-graph related questions. take a look at stack, other data structures, and maybe other domains like Biology